messaging.rst

```diff
@@ -610,31 +609,34 @@ Preconditions
 Fields
 ^^^^^^
 
+``WithdrawExpired`` is an :term:`onchain message`. This is the format in which it is packed to
+compute the ``signature``, and in which it could be forwarded to a contract.
```

Comment:
None of the fields of `WithdrawExpired` ever go to the contract. Which raises the question why the signature is constructed in the way that the contracts expect. Is this just for consistency with the other two withdraw messages, or is there a different reason @rakanalh?